src/Workspaces/Core/Portable/Classification/Classifier.cs

```diff
@@ -5,35 +5,27 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.Extensions;
-using Microsoft.CodeAnalysis.Shared.Extensions;
 using Microsoft.CodeAnalysis.Text;
 using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.Classification
 {
+    /// <summary>
+    /// 
```

Comment:
You know it.

---
In reply to: [78663860](https://github.com/dotnet/roslyn/pull/13741#discussion_r78663860) [](ancestors = 78663860)